packages/cli/__tests__/zosfiles/__unit__/compare/ds/Dataset.handler.unit.test.ts

```diff
@@ -178,6 +178,41 @@ describe("Compare data set handler", () => {
             expect(getDiffStringSpy).toHaveBeenCalledWith("compared", "compared", options);
         });
 
+        it("should compare two data sets containing carriage returns in terminal with --seqnum specified(Windows Specific)", async () => {
+            if(process.platform === "win32")
```

Comment:
should we do `(process.platform === "win32" ? it : it.skip)("should compare ...` ?
That way we don't have an empty test on other operating systems?

I don't recall if this is the exact syntax though.